BUILD.md

```diff
@@ -125,7 +125,7 @@ The complete list of build options can be found by running `./build.sh (or .\bui
 ### CUDA
 #### Pre-Requisites
 * Install [CUDA](https://developer.nvidia.com/cuda-toolkit) and [cuDNN](https://developer.nvidia.com/cudnn)
-  * ONNX Runtime is built and tested with CUDA 10.0 and cuDNN 7.6 using the Visual Studio 2017 14.11 toolset (i.e. Visual Studio 2017 v15.3). CUDA versions from 9.1 up to 10.1, and cuDNN versions from 7.1 up to 7.4 should also work with Visual Studio 2017.
+  * ONNX Runtime is built and tested with CUDA 10.1 and cuDNN 7.6 using the Visual Studio 2019 14.12 toolset (i.e. Visual Studio 2019 v16.5). CUDA versions from 9.1 up to 10.1, and cuDNN versions from 7.1 up to 7.4 should also work with Visual Studio 2017.
```

Comment:
Did you mean to say 2019 here? "should also work with Visual Studio 2017"

Reply:
I am not sure if CUDA 9.1 would work with VS 2019, they should work with the latest VS 2017 version though. Is that statement confusing ? We can add it to another bullet point. Basically, I think we want to say prior versions of CUDA will also gel with ORT, but I am not sure which VS version is required...

Reply:
We can just say that we also support cuda version 9.1 up to 10 and cudnn versions  7.1 up to 7.4 and leave the mention of the compiler.

Reply:
Sure, fixed.